### ISSUE TITLE
child/dependent care expenses on puf.csv vs cps.csv

### ISSUE THREAD

**Issue by MattHJensen** [open-source-economics/taxdata#116]
I see from our tax-calculator documentation that we have child/dependent care expenses for qualifying persons from form 2441 (e32800) on the taxdata_cps file. Is that from the CPS directly or is it imputed? Do we have values for filing units that don’t claim the credit (because they don’t have income tax liabilities or for other reasons)?

A potential tax-calculator user is interested in analyzing the CDCTC under current law and expansion reforms, and I am wondering if they can safely do that with the cps file. My understanding is that he could not analyze expansions with the puf.csv file. 

@andersonfrailey

**Comment by martinholmer** [comment]
I have no idea about the source of the `e32800` data in the `cps.csv` file, so I'll leave that key question to @andersonfrailey.

However, the documentation does appear to be correct.  Here are some **unweighted** tabulations of the `cps.csv` and `puf.csv` files:
```
$ ../csvvars cps.csv | grep e32800
30 e32800
$ awk -F, 'NR>1{t++;if($30>0)n++}END{print t,n,100*n/t}' cps.csv
456465 35701 7.82119

$ ../csvvars puf.csv | grep e32800
49 e32800
$ awk -F, 'NR>1{t++;if($49>0)n++}END{print t,n,100*n/t}' puf.csv
215525 9045 4.19673
```
So, nearly 8% of the CPS filing units have positive `e32800` amounts (from somewhat).
In comparison, on slightly more the 4% of PUF filing units have positive `e32800` amounts from Form 2441.

@MattHJensen

**Comment by andersonfrailey** [comment]
@MattHJensen `e32800` on the CPS is imputed so there should be values even for those who don't ultimately claim the credit.

**Comment by intownllc** [comment]
Thanks @andersonfrailey. How is it imputed? What do you think would be the effortnlevel for someone to duplicate the imputation for the puf.csv file?

**Comment by MattHJensen** [comment]
Thanks @andersonfrailey. Is there anything you can point me to read about how it is inputed? What do you think would be the effort level for someone to duplicate the imputation for the puf.csv file?

**Comment by andersonfrailey** [comment]
@MattHJensen a short overview of the imputation process can be found [here](https://github.com/open-source-economics/taxdata/blob/master/cps_data/cps_file_doc.md). I just noticed that I forgot to add a paragraph specifically for child/dependent care expenses, but they're imputed in the same manor as most of the other variables. First a legit model is used to determine which tax units see who has any expense, then a standard OLS model is used to determine the actual value.

It's difficult to estimate an effort level to duplicate the imputation on the puf.csv file. The parameters for both models are drawn from the raw SOI PUF, so I'm not sure how effective they would be for imputing values onto the puf.csv. It would likely be better to find another approach to it.

**Comment by martinholmer** [comment]
In the discussion of issue taxdata #116, @andersonfrailey said:
> [A] short overview of the imputation process can be found here.  I just noticed that I forgot to add a paragraph specifically for child/dependent care expenses, but they're imputed in the same manor as most of the other variables [that have their imputation equations estimated using PUF data].  First a logit model is used to determine which tax units see who has any expense, then a standard OLS model is used to determine the actual value.

But because the PUF data has care expenses for **only** those who file a tax return that includes a Form 2441, the imputation equations estimated using those PUF data are biased.  So, for example, the fraction of families in the whole population with positive care expenses is much larger than the fraction of families who submit a tax return that includes a positive amount on Form 2441.

If the care expense imputation equations had been estimated with other (representative) data, then all would be good.  But the `cps.csv` data on care expenses is just as biased as the care expense data in the `puf.csv` file.

@MattHJensen

**Comment by MattHJensen** [comment]
@andersonfrailey and @martinholmer, thanks for your feedback on this issue. My conclusion is that we don't currently have a data source suitable for analyzing expansions to the CDCTC. Closing.

**Comment by martinholmer** [comment]
@MattHJensen said in closed taxdata issue #116:
> My conclusion is that we don't currently have a data source suitable for analyzing expansions to the CDCTC.

Given this conclusion, should we add warnings to Tax-Calculator when users try to increase the child/dependent care credit amount?

**Comment by MattHJensen** [comment]
@martinholmer said:

> Given this conclusion, should we add warnings to Tax-Calculator when users try to increase the child/dependent care credit amount?

Yes. I will do that now.
